backend/apps/auth/src/routes/auth.py

```
@@ -0,0 +1,32 @@
+    token = credentials.credentials
+    try:
+        payload = verify_jwt_token(token)
+        return {
+            "user_id": payload["user_id"],
+            "email": payload["email"]
+        }
+    except:
+        raise HTTPException(401, detail="Invalid token")
+
+@router.get("/me")
+def get_me(credentials = Depends(security)):
```

Ruff (0.14.1)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)